pkg/utils/ipset.go

```diff
@@ -225,18 +236,18 @@ func (set *Set) Add(addOptions ...string) (*Entry, error) {
 // Del an entry from a set. If the -exist option is specified and the entry is
 // not in the set (maybe already expired), then the command is ignored.
 func (entry *Entry) Del() error {
-	_, err := entry.Set.Parent.run(append([]string{"del", entry.Set.Name}, entry.Options...)...)
+	_, err := entry.Set.Parent.run(append([]string{"del", entry.Set.name()}, entry.Options...)...)
 	if err != nil {
 		return err
 	}
-	entry.Set.Parent.Save()
+	entry.Set.Parent.Save() // <-- EKM: Implications??
```

Comment:
Is this comment required?

Reply:
No. Sorry I forgot to remove it, sorry. It was a personal note.
The Save function will add *all* sets to the IPSet object. Eventually all IPSet objects may inlcude all sets. Something must be done eventually but it obviously works and I coudn't make an analyze fast so I put the reminder there.